code/datums/setup_option/backgrounds/origin_upbringing.dm

```diff
@@ -162,10 +162,10 @@
 
 	stat_modifiers = list(
```

Comment:
Why are you nuking the savant?

Reply:
Because unlike Klutz, that gives the `CLUMSY` mutation, this one has absolutely zero mechanical drawbacks and it was a lot of stats with just COG as a tradeoff, I'll at least nerf it if it's better